index.html

```diff
@@ -27,19 +27,19 @@ <h1>My Projects</h1>
       <div id="main-con">
         <div id="project-tile">
           <a href="#"> <img src="images/p1.png" alt="" /></a>
-          <div id="title">project 1</div>
+          <div id="title">project 11</div>
         </div>
         <div id="project-tile">
           <a href="#"> <img src="images/p2.png" alt="" /></a>
-          <div id="title">project 2</div>
+          <div id="title">project 12</div>
```

Comment:
Duplicate use of ID tag. The project tiltle ID is enough. Eg. 
 <div id="project-tile">
          <a href="#"> <img src="images/p2.png" alt="" /></a>
          <div >project 2</div>
          <div >project 12</div>
Remove repetition of ID in the child elements.